index.html

```diff
@@ -112,212 +113,247 @@ <h4 class="grey-text">Back End Dev</h4>
                 <h4 class="grey-text">2015</h4>
               </div>
               <p class="desc-text">
-                A daily selection of privately <br />
-                personalized reads; no accounts or <br />
-                sign-ups required.
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
               </p>
-              <div class="work-tech">
+              <div class="work-tech small-ul-spacing">
                 <ul>
                   <li>html</li>
                   <li>css</li>
                   <li>javascript</li>
                 </ul>
               </div>
             </div>
+            <div class="see-work">
+              <button>See Project</button>
+            </div>
           </div>
-          <div class="see-work">
-            <button>See Project</button>
-            <!-- <a href="#">See Project</a> -->
           </div>
-        </div>
       </div>
 
       <div class="works-wrapper">
         <div class="works-section">
-          <div class="work-snapshot">
-            <img
+          <div class="work-snapshot swap">
+            <img class="mobile-pic"
               src="assets/works-snapshot/Snapshoot Portfolio 2.png"
               alt="work-snapshot-1"
             />
+            <img class="desktop-pic"
+            src="assets/desktop/Desktop-work-snapshot-2.png"
+            alt="Desktop-work-snapshot-2"
+            />
           </div>
           <div class="work-description">
             <div class="desc-wrapper">
               <h2>Multi-Post<br />Stories</h2>
               <div class="dec-details">
-                <h4>CANOPY</h4>
+                <h4 class="mobile-text">CANOPY</h4>
+                <h4 class="desktop-text">FACEBOOK</h4>
                 <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot" />
-                <h4 class="grey-text">Back End Dev</h4>
+                <h4 class="grey-text mobile-text">Back End Dev</h4>
+                <h4 class="grey-text desktop-text">Full Stack Dev</h4>
```

Comment:
- [ optional ] As suggested by the previous reviewer Kindly make this text in one line and not broken in 2 lines to improve readability and to follow the design in Figma 💯.



The Figma design Look likes this  : 

![Capture](https://user-images.githubusercontent.com/93598189/208060595-627cedfb-822a-4986-ad3b-aba63301fe42.PNG)


Your Design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/208059521-2c83e245-274b-4dec-b039-b04da524617d.PNG)